Core/Core/Planner/CreateTodoViewController.swift

```diff
@@ -0,0 +1,178 @@
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+
+public class CreateTodoViewController: UIViewController, ErrorViewController {
+
+    @IBOutlet weak var titleLabel: DynamicTextField!
+    @IBOutlet weak var dateTitleLabel: DynamicLabel!
+    @IBOutlet weak var dateTextField: DynamicTextField!
+    @IBOutlet weak var descTextView: UITextView!
+    @IBOutlet weak var courseTitleLabel: DynamicLabel!
+    @IBOutlet weak var courseSelectionLabel: DynamicLabel!
+    @IBOutlet weak var courseChevron: IconView!
+    @IBOutlet weak var selectCourseButton: UIButton!
+
+    let env = AppEnvironment.shared
+    let datePicker = UIDatePicker()
+    var createPlannerNote: Store<CreatePlannerNote>?
+    var selectedDate: Date = Clock.now
+    var selectedCourseName: String? {
+        guard let c = selectedCourse else { return NSLocalizedString("None", comment: "")  }
+        return c.name
+    }
+    var selectedCourse: Course?
+    var formattedDate: String {
+        DateFormatter.localizedString(from: selectedDate, dateStyle: .medium, timeStyle: .short)
+    }
+
+    public static func create() -> CreateTodoViewController {
+        let vc = loadFromStoryboard()
+        return vc
+    }
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        title = NSLocalizedString("New To Do", comment: "")
+        titleLabel.placeholder = NSLocalizedString("Title...", comment: "")
+        dateTitleLabel.text = NSLocalizedString("Date", comment: "")
+        if descTextView.responds(to: #selector(getter: UITextField.placeholder)) {  // without this check, it fails unit tests
+            descTextView.setValue(NSLocalizedString("Description", comment: ""), forKey: "placeholder")
+        }
+        descTextView.font = UIFont.scaledNamedFont(.regular16)
+        dateTextField.text = formattedDate
+        courseChevron.tintColor = UIColor.named(.borderMedium)
+        courseChevron.image = UIImage.icon(.arrowOpenRight, .solid)
+        courseSelectionLabel.text = selectedCourseName
+        courseSelectionLabel.textColor = UIColor.named(.textDark)
+
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: NSLocalizedString("Cancel", comment: ""), style: .plain, target: self, action: #selector(actionCancel))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: NSLocalizedString("Done", comment: ""), style: .plain, target: self, action: #selector(actionDone))
```

Comment:
These & other NSLocalizaedString calls in Core need to have the `bundle: .core` argument, or they won't get properly localized.